frontend/src/components/public_group/AmountPicker.js

```diff
@@ -44,7 +44,7 @@ export default class AmountPicker extends React.Component {
         )}
 
         <div className='max-width-1 mx-auto' style={{position: 'absolute', left: '0', right: '0', margin: '-25px auto'}}>
-          {!hasStripe && (
+          {(!hasStripe || (hasPaypal && hasStripe)) && (
```

Comment:
In the context of the website, hasPaypal should refer to whether or not this collective accepts donations via PayPal. 

What you are talking about should only apply to the app. That's the only place where there is the concept of paying out using PayPal.